projects/step-core/src/lib/modules/table/components/table/table.component.scss

```diff
@@ -1,10 +1,30 @@
-:host {
-  position: relative;
-  display: block;
+step-table {
+  height: 100%;
+  display: flex;
+  flex-direction: column;
 
-  table {
-    width: 100%;
-    border-spacing: 10px 0 !important;
+  .table-container {
+    flex: 1;
+    overflow: auto;
+    display: block;
+
+    table {
+      width: 100%;
+      border-collapse: separate;
+
+      thead tr {
+        position: sticky;
```

Comment:
There is no information in the ticket's description, that sticky headers should be implemented. Why it was added?

Reply:
better UX

Reply:
I actually like it. We just have to be careful in places where there could be multiple tables, but as far as I saw all works well!